src/System.Collections/tests/Generic/List/ForEach_Action.cs

```diff
@@ -107,6 +107,8 @@ public void CallForEach(T[] items, Action<T> action)
             for (int i = 0; i < items.Length; ++i)
                 list.Add(items[i]);
 
+            Assert.Throws<ArgumentNullException>("action", () => list.ForEach(null));
+
```

Comment:
Ah ok,I wasn't aware of that. I thought Jenkins might be only looking at my latest commit so I squashed.

Is that why there's no `List <T>` in the future branch?